src/index.js

```diff
@@ -1,64 +1,25 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable no-console */
-import express from 'express';
-import { urlencoded, json } from 'body-parser';
-import cors from 'cors';
-import errorhandler from 'errorhandler';
-import router from './routes/index';
-import ErrorHandler from './utils/error';
+import app from './app';
 import logger from './utils/winston';
 
-const isDevelopment = process.env.NODE_ENV;
-
-const app = express();
-
-app.use(cors());
-
-app.use(urlencoded({ extended: false }));
-app.use(json());
-
-// used in testing heroku
-app.get('/', (req, res) => res.status(200).json({
-  message: 'Welcome to Barefoot Nomad'
-}));
-
-// to throw an error to the logger
-app.get('/err', (req, res) => {
-  throw new ErrorHandler('Errror here');
+app.set('port', process.env.PORT || 3000);
+const server = app.listen(app.get('port'), () => {
+  // eslint-disable-next-line no-console
+  console.log(`Express running → PORT ${server.address().port}`);
```

Comment:
Use the `logger.info`  here. You can remove the eslint disable comment after that.